.github/workflows/release.yml

```diff
@@ -1,73 +1,71 @@
-            archive: tar.gz
-
+            bin: crunch64.exe
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v4
-
-      - name: Compile cli for artifact
-        id: compile_cli
-        uses: rust-build/rust-build.action@v1.4.4
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
         with:
-          RUSTTARGET: ${{ matrix.target }}
-          ARCHIVE_TYPES: ${{ matrix.archive }}
-          SRC_DIR: cli/
-          UPLOAD_MODE: none
-
-      - name: Upload cli artifact
-        uses: actions/upload-artifact@v3
+          targets: ${{ matrix.target }}
+      - uses: Swatinem/rust-cache@v2
+      - name: Build
+        run: |
+          if [ "${{ matrix.os }}" == "windows-latest" ]; then
+            cargo build --release --manifest-path cli/Cargo.toml
+          else
+            cargo build --release --manifest-path cli/Cargo.toml --target ${{ matrix.target }}
+          fi
+      - name: Create archive
+        run: |
+          folder=crunch64-v${{ github.ref_name }}-${{ matrix.target }}
+          mkdir -p $folder
+          if [ "${{ matrix.os }}" == "windows-latest" ]; then
+            copy cli\\target\\release\\${{ matrix.bin }} $folder\\
+            copy README.md LICENSE $folder\\
+            7z a $folder.zip $folder
+          elif [ "${{ matrix.os }}" == "macos-latest" ]; then
+            cp cli/target/${{ matrix.target }}/release/${{ matrix.bin }} $folder/
+            cp README.md LICENSE $folder/
+            zip -r $folder.zip $folder
+          else
+            cp cli/target/release/${{ matrix.bin }} $folder/
+            cp README.md LICENSE $folder/
+            tar -czvf $folder.tar.gz $folder
+          fi
+      - uses: actions/upload-artifact@v4
         with:
-          name: crunch64-cli-${{ matrix.target }}
-          path: |
-            ${{ steps.compile_cli.outputs.BUILT_ARCHIVE }}
-            ${{ steps.compile_cli.outputs.BUILT_CHECKSUM }}
+          name: ${{ matrix.target }}
+          path: crunch64-v${{ github.ref_name }}-${{ matrix.target }}.*
 
-      - name: Compile and release cli
-        uses: rust-build/rust-build.action@v1.4.4
-        if: startsWith(github.ref, 'refs/tags/')
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          RUSTTARGET: ${{ matrix.target }}
-          ARCHIVE_TYPES: ${{ matrix.archive }}
-          SRC_DIR: cli/
-          UPLOAD_MODE: release
-  publish:
+  release:
+    needs: build
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v4
-
-      - name: Setup Rust toolchain
-        uses: actions-rs/toolchain@v1
+      - uses: actions/download-artifact@v4
         with:
-          toolchain: stable
-          override: true
-
-      - name: Publish crate
-        uses: katyo/publish-crates@v2
-        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
So we no longer release the crate to crates.io?

Reply:
The `katyo/publish-crates@v2` action is the one that actually does the publishing, but I couldn't see it anywhere, nor an equivalent, in the new file. Maybe I missed it.